aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/DeleteHandler.java

```diff
@@ -80,11 +63,4 @@ public DeleteHandler(final HandlerConfig config) {
                 ))
                 .done((deleteRequest, deleteResponse, proxyInvocation, model, context) -> ProgressEvent.defaultSuccessHandler(null));
     }
-
-    private boolean isDbInstanceDeletingException(final Exception exception) {
-        if (exception instanceof AwsServiceException) {
-            return Optional.ofNullable(exception.getMessage()).orElse("").contains(DB_INSTANCE_IS_BEING_DELETED_ERR);
-        }
-        return false;
-    }
```

Comment:
I think we will need to make ErrorRuleSet use errorMessage substring comparing in decision making in the future.
RdsClient can throw same error code but with different message which may lead to different behavior in handler.

Reply:
Good point, will submit a separate PR. Thanks for the hint!